docs/source/dev/supported_vrplib_fields.rst

```diff
@@ -63,6 +63,9 @@ PyVRP supports the following data sections:
       Array of backhaul quantities, one for each location.
       This is the amount picked up at the client and transported back to the depot.
 
+   ``CAPACITY_SECTION``
+      Maximum capacity for each vehicle.
```

Comment:
Question: should we say something about how this can both be a section and a specification? And, presumably, that if you pass both, one of them has priority?

Reply:
Done!